overlord/snapstate/snapstate.go

```diff
@@ -1071,6 +1094,62 @@ func InstallWithDeviceContext(ctx context.Context, st *state.State, name string,
 	return doInstall(st, &snapst, snapsup, 0, fromChange, nil)
 }
 
+func InstallPathMany(ctx context.Context, st *state.State, paths []string, sideInfos []*snap.SideInfo, userID int, flags *Flags) ([]*state.TaskSet, error) {
```

Comment:
we should swap sideInfos and paths to match InstallPath, no?

Reply:
Sure, I swapped it yesterday because path first seemed more "natural" but it's inconsistent with InstallPath. I'll change it